client/src/components/HostId.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import HostPhoto from './HostPhoto';
+import HostDetails from './HostDetails';
+import styled from 'styled-components';
```

Comment:
Is this used here?

Reply:
Not yet! All of the styling will come in a bit. In future PRs, I'm going to do all of the build, style and test for one component in one PR. More "feature-like" process rather than laying the groundwork for many different features at once.